__tests__/integration/previewSaveRDF.test.js

```diff
@@ -27,6 +28,13 @@ describe('Previewing the RDF', () => {
     await pupExpect(page).toMatch('Which group do you want to save to?')
     await pupExpect(page).toMatch('Which group do you want to associate this record to?')
 
+    // test to make sure select list has sorted list of groups from config?
+    const groupsForSaving = await page.$$eval('form.group-select-options select option', nodes => nodes.map(n => [n.value, n.innerText]))
+    const expectedGroups = Object.entries(Config.groupsInSinopia)
```

Comment:
@jcoyne, yeah, fair point.

i'm not really sure, though.  the thing i wanted to test is that all of the groups we care about get displayed in alphabetical order.  since the filtering and alphabetizing happens in `Editor`, but the display happens in `GroupChoiceModal`, i wasn't sure how to test that.

i guess maybe i could try to write a unit test for `Editor` that makes sure the right list gets passed to `GroupChoiceModal`?  i do sorta like having an integration test that gets at what's being done here, but i don't feel that strongly about it, and displaying the right list in the right order is what i was really trying to get at here, one way or the other.

how strong a pref do you have about this?  like what would be your pref as far as blocking merge of this PR to turn this test into a unit test?

Reply:
Well, you can see how big the test already grew (as you need to rebase it).  It's already super complicated.  I'd be pretty satisfied by just a unit test for Editor.

Reply:
I'd suggest possibly creating a separate integration test file with just this one test??  Also, maybe some of the pupExpect statements could change to waitForSelector so they are clearly more about setup than test expectations?

Reply:
@jcoyne and @ndushay, i just took the idea of the integration test and moved it over to `GroupChoiceModal.test.js`, which, as luck would have it, was much easier in light of @jcoyne's #748.  that felt slightly more satisfying to me than testing what `Editor` passed along.

ended up just backing out of the integration test.